lib/KeyboardAwareMixin.js

```diff
@@ -1,13 +1,15 @@
 /* @flow */
 
 import { PropTypes } from 'react'
-import ReactNative, { TextInput, Keyboard } from 'react-native'
+import ReactNative, { TextInput, Keyboard, Platform } from 'react-native'
 import TimerMixin from 'react-timer-mixin'
 
 const _KAM_DEFAULT_TAB_BAR_HEIGHT = 49
 const _KAM_KEYBOARD_OPENING_TIME = 250
 const _KAM_EXTRA_HEIGHT = 75
 
+const isAndroid = () => Platform.OS === 'android'
```

Comment:
Can you change this to:

```js
const isAndroid = Platform.OS === 'android'
```

It's not necessary to run the function every time you need to check if it's Android or not.